monitor_check.py

```diff
@@ -127,8 +131,49 @@ def p(*args, **kwargs):
         message += ' \N{FIRE}'
     p(message, fg=fg)
 
+    if browser_lock and not bz and content_length > LIMIT:
+        await open_bz(package, build, status, browser_lock)
+
+
+async def open_bz(package, build, status, browser_lock):
+    summary = f"{package} fails to build with Python 3.8"
+
+    description = dedent(f"""
+        {package} fails to build with Python 3.8.0b1.
+
+        This report is automated and not very verbose, but we'll try to get back here with details.
+
+        For the build logs, see:
+        https://copr-be.cloud.fedoraproject.org/results/@python/python3.8/fedora-rawhide-x86_64/{build:08}-{package}/
+
+        For all our attempts to build {package} with Python 3.8, see:
+        https://copr.fedorainfracloud.org/coprs/g/python/python3.8/package/{package}/
```

Comment:
Testing and mass rebuild of packages is happening in copr. You can follow these instructions to test locally in mock if your package builds with Python 3.8: https://copr.fedorainfracloud.org/coprs/g/python/python3.8/